beetmoverscript/script.py

```diff
@@ -153,9 +154,16 @@
         log.warning("Destination {} already exists with {} keys".format(
                     releases_prefix, len(releases_keys_checksums)))
 
-    # Weed out RELEASE_EXCLUDE matches
+    # Weed out RELEASE_EXCLUDE matches, but allow partners specified in the payload
+    push_partners = context.task['payload'].get('partners', [])
     for k in candidates_keys_checksums.keys():
-        if not matches_exclude(k, RELEASE_EXCLUDE):
+        partner_match = get_partner_match(k, push_partners)
+        if partner_match:
```

Comment:
Should this check some subset of `RELEASE_EXCLUDE` that avoids uploading too many things? Or are we precise about our partner artifacts?

Reply:
You're right, this could do with some tightening up.

Reply:
I've modified this code so that it does a quick check to see if the key is a possible partner, then checks if the key starts with candidates_dir + 'partner-repack' + any of the publishing partners + 'v1'